dashboard/app/models/user.rb

```diff
@@ -733,9 +733,16 @@ def primary_script
   end
 
   def needs_to_backfill_user_scripts?
-    user_scripts.empty? && !user_levels.empty?
+    user_scripts.empty? &&
+      !user_levels.empty? &&
+      # Backfill only applies to users created before UserScript model was introduced.
+      created_at < Date.new(2014, 9, 11)
```

Comment:
pushed by code-dot-org/website-ci#259 on 9/10 PDT (which was actually 9/11 UTC). i'll bump it to 9/15 to be safe.